### ISSUE TITLE
Add support for badges on top TabFolder bar

### ISSUE THREAD

**Issue by mpost** [eclipsesource/tabris-js#2031]
### Problem description

At the moment the `Tab` badge is only shown on a `TabFolder` with `tabBarLocation` set to `'bottom'`. At least on Android the [material design guidelines](https://material.io/develop/android/components/tab-layout/) allow to have a top badge as well.

![image](https://user-images.githubusercontent.com/73655/79764585-d6dc1e80-8325-11ea-8dcb-755329a6e82f.png)

### Expected behavior

Display badges on the top `TabFolder` bar.

### Environment

- Tabris.js version: 3.4

### Code snippet

```jsx
import {contentView, Tab, TabFolder, TextView} from 'tabris';

contentView.append(
  <TabFolder stretch>
    <Tab title='Cart' badge={5}>
      <TextView center text='Content of Tab Cart'/>
    </Tab>
  </TabFolder>
);
```

**Comment by mpost** [comment]
After further research it turned out that the iOS platform does not support the top badge. We could therefore implement it as an Android only feature.

**Comment by mpost** [comment]
Implemented for the 3.5 release via the native platform and https://github.com/eclipsesource/tabris-js/commit/2262b6ce83f056de0663b6e46a52536537f9f03d
